pkgs/development/tools/misc/lsof/default.nix

```diff
@@ -22,12 +25,14 @@ stdenv.mkDerivation rec {
   };
 
   unpackPhase = "tar xvjf $src; cd lsof_*; tar xvf lsof_*.tar; sourceRoot=$( echo lsof_*/); ";
+  
+  patches = [ ./dfile.patch ];
 
-  preBuild = "sed -i Makefile -e 's/^CFGF=/&	-DHASIPv6=1/;';";
+  preBuild = "sed -i Makefile -e 's/^CFGF=/&	-DHASIPv6=1/;' -e s/-lcurses/-lncurses/ ;";
```

Comment:
@Mic92, I found the same thing. Unless postConfigure is turned on, preBuild seems to be the only reasonable place for this line.